### DISCUSSION TITLE
Welcome to jtxBoard Discussions!

### DISCUSSION THREAD

**Discussion by patrickunterwegs** [TechbeeAT/jtxBoard#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ronane** [comment]
Hello,

just here to thank you for jtxBoard, I discovered the app through davx5 and that's a nice log/note/task app.

**Reply by patrickunterwegs** [reply]
Hey, thanks for the feedback 🙂

**Comment by DickinsonSa** [comment]
I've been looking for the perfect app for a long time - this one ticks the most boxes so far. I'm happy to contribute in some small way as a 'thank you' for all the hard work.

**Reply by patrickunterwegs** [reply]
Thanks @DickinsonSa , I'm looking forward to your feedback and contributions 😊

**Comment by ThureHauser** [comment]
Hi, i am new to jtx and its functions.
Is it possible to add tasks from or by email?
In some email clients i can convert an email to a task, but not in the kmail i try out now. In other tasklist software, i can ad tasks by sending an email to a specific task list address, but that is not possible in kcalendar, i also try out.
I hope you can help me 🙂

**Reply by patrickunterwegs** [reply]
Hi @ThureHauser , 
jtx Board is only client and is not integrated with e-mail or calendar software. Other task list software might be integrated server solutions where this is possible. However, in jtx Board you can't create a task by sending it to a specific address, this would be a functionality that needs to be provided by your CalDAV server provider. 
But what you can do is marking a text in any application on your phone, click on share and select jtx Board. The app will then give you the option to create a new journal, task or note based on the text that your shared with the app!

PS: Please start new discussions in a dedicated topic

**Reply by ThureHauser** [reply]
Thank you for the great tip!
